src/components/SuperTiles.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import CodeStreamTile from './CodeStreamTile';
+import GuidedInstallTile from './GuidedInstallTile';
+
+const SuperTiles = () => {
```

Comment:
You could do an implicit return here if you wanna knock a few lines out of the component, nbd either way, just a little cleaner:

`const SuperTiles = () => (
    <div
      css={css`
        display: grid;
        margin-bottom: 16px;
        gap: 16px;
        grid-template-columns: repeat(2, 1fr);
        @media (max-width: 1180px) {
          grid-template-columns: repeat(1, 1fr);
        }
      `}
    >
      <GuidedInstallTile />
      <CodeStreamTile />
    </div>
  );`